### PR TITLE
Apply minor fixes before the v3.3.0 release

### PR DESCRIPTION
**Pull request by PawelLipski**

### CODE REVIEW

File: frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/TraverseFromCurrentAction.java [hunk 1]
```diff
@@ -12,14 +10,4 @@ public class TraverseFromCurrentAction extends BaseTraverseAction {
   public @Nullable String getNameOfBranchUnderAction(AnActionEvent anActionEvent) {
     return getCurrentBranchNameIfManaged(anActionEvent);
   }
-
-  @Override
-  @UIEffect
-  protected void onUpdate(AnActionEvent anActionEvent) {
```
Comment (PawelLipski) [inline frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/TraverseFromCurrentAction.java:18]: Assuming it's redundant indeed, https://github.com/VirtusLab/git-machete-intellij-plugin/pull/1397#discussion_r1056356695

Comment (mkondratek) [reply frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/TraverseFromCurrentAction.java:18]: oh, probably you are right -
- [ ] pls just make sure that you run ide locally, and the action looks correctly from the toolbar and from the find action dialog

**Review by mkondratek — CHANGES_REQUESTED** [review]

**Review by mkondratek — APPROVED** [review]
but pls check and tick the box ☝️

File: frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/TraverseFromCurrentAction.java (hunk 1 above)
Comment (PawelLipski) [reply frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/TraverseFromCurrentAction.java:18]: From toolbar: looks perfect

From find action dialog: looks that no branch-specific action is enabled 😯 

image

image

image


All greyed out... compare to the actions that are available:

image

Is it even expected? can you see the same in your IDE? Lemme confirm this is also the case with current `master`...

Comment (PawelLipski) [reply frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/TraverseFromCurrentAction.java:18]: Woooow there's a lot of mess 😅 

image

Comment (PawelLipski) [reply frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/TraverseFromCurrentAction.java:18]: The reason for Pull action being disabled is clearly wrong... and similarly with many other places. Not a thing of high priority to be fixed right now... lemme open a separate issue

Comment (PawelLipski) [reply frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/TraverseFromCurrentAction.java:18]: #1401

Comment (mkondratek) [reply frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/TraverseFromCurrentAction.java:18]: I see some descriptions are wrong but in general the selected-based actions shall by disabled from the find action dialog

Comment (PawelLipski) [reply frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/TraverseFromCurrentAction.java:18]: Yeah... but this specific problem is about _Pull **current** branch_ action, not **selected** branch 🤔

**Pull request merged**